ui/src/shared/components/cloud/CloudOnly.scss

```diff
@@ -1,20 +1,67 @@
-.dashboard--rate-alert {
-    padding: 0 $page-gutter;
-    width: 100%;
-    padding-bottom: 16px;
+.load-data--asset-alert {
+    margin-bottom: $cf-marg-d;
 }
 
-.load-data--rate-alert {
-    padding: 0 $page-gutter;
-    padding-bottom: 16px;
-    max-width: 1608px;
+.cf-page-control-bar {
```

Comment:
unrelated to code: why are changes sometimes highlighted?

Reply:
It usually highlights it if the entire line/block didn't change but some parts of it did.